src/provider/sequencer.ts

```diff
@@ -225,7 +225,7 @@ export class SequencerProvider implements ProviderInterface {
       if (endpoint === 'estimate_fee') {
         return parseAlwaysAsBig(textResponse, (_, v) => {
```

Comment:
Noticed one more minor thing, with the switch to BigInts the second argument can be removed.

Reply:
Nice catch. Thanks